src/components/AppSidebar/AppSidebar.vue

```diff
@@ -39,7 +39,7 @@
 				<!-- sidebar details -->
 				<div :class="{ 'app-sidebar-header__desc--with-star': canStar }" class="app-sidebar-header__desc">
 					<!-- favourite icon -->
-					<a v-if="canStar" :class="isStarred ? 'icon-starred' : 'icon-star'"
+					<a v-if="canStar" :class="[ isStarred ? 'icon-starred' : 'icon-star', { loading: starLoading } ]"
```

Comment:
`icon-loading-small` would look better. And the `loading` class is deprecated :wink:

Reply:
Oh, I didn't know that `loading` is deprecated. Is this documented somewhere?

Please have a look at the new commit. :-)

Reply:
Well, I cannot find it, so I may have dreamed of doing it ^^'